features/facebookImport/test/datasets/connected-advertisers-data.js

```diff
@@ -12,19 +12,23 @@ export function wrapConnectedAdvertisersData(data) {
     return { [CONNECTED_ADVERTISERS_DATA_KEY]: data };
 }
 
+export const CONNECTED_ADVERTISERS_DATA_SPECIFICATION = [
```

Comment:
Instead of repeating it, can we just use some common JSON?

Reply:
Defined it only in one place